governance/fma-isthmus-l2-pectra.md

```diff
@@ -0,0 +1,164 @@
+
+- **Description:** The chain could halt if a network upgrade transaction can't be included in the upgrade block.
+- **Risk Assessment:** High severity, low likelihood
+- **Mitigations:**
+  1. end-to-end tests created for network upgrade transactions
+  2. same upgrade path as previous predeploys
+- **Detection:** Monitors should trigger an alert immediately
+- **Recovery Path(s)**: This would require an emergency update if it occurred fixing the network upgrade transaction and updates to `op-node`.
+
+### FM2: Chain halt due to request hash mismatch
+
+- **Description:** Requests hash was added to the block header. If it mismatches between geth and op-node, the chain could halt
+- **Risk Assessment:** High severity, low likelihood
+- **Mitigations:**
+  1. End-to-end tests
+  2. Simple validation rules - always must be an empty hash
+- **Detection:** Monitors should trigger an alert immediately.
+- **Recovery Path(s)**: This would require an emergency update if it occurred fixing the network upgrade transaction and updates to `op-node`.
+
+### FM3: EIP-7702 transactions cannot be included on the chain
+
+- **Description:** SetCode (EIP-7702) transactions can't be included on chain.
+- **Risk Assessment:** Low impact, low likelihood
+- **Mitigations:** 
+	1. End-to-end tests of set code transactions before and after the fork
+- **Detection:** Sending set code transactions would fail; probably manual detection is most likely.
+- **Recovery Path(s)**: This would require another update and fork.
+
+### FM4: BLS Precompiles could cause increased FP Program execution time
+
+- **Description:** If implemented in the FP program, BLS precompiles could greatly increase computation time.
+- **Risk Assessment:** High impact, low likelihood
+- **Mitigations:**
+	1. Fully mitigated by using an accelerated precompile that calls out to L1 instead of calculating inside the program.
+- **Detection:** op-program trace runner computation time will increase (possibly triggering an alert)
+- **Recovery Path(s)**: If this occurred, which it can't with the mitigations, we'd have to make op-program efficient enough to execute
+BLS precompiles in sufficient time.
+
+### FM5: BLS Precompiles could fail to execute in the FP program
+
+- **Description:** If implemented in the FP program, BLS precompiles could fail to execute. This could cause certain blocks to be unprovable.
+- **Risk Assessment:** High impact, low likelihood
+- **Mitigations:**
+	1. Fully mitigated by using an accelerated precompile that calls out to L1 instead of calculating inside the program. This means it should 
+  have the same performance characteristics as existing crypto precompiles like `ecrecover`.
+- **Detection:** op-program trace runner failure (possibly triggering an alert)
+- **Recovery Path(s)**: If this occurred, which it can't with the mitigations, we'd have to make op-program efficient enough to execute BLSprecompiles in sufficient time.
```

Comment:
nit
```suggestion
- **Recovery Path(s)**: If this occurred, which it can't with the mitigations, we'd have to make op-program efficient enough to execute BLS precompiles in sufficient time.
```

Reply:
fixed